### PR TITLE
feat: featured suggestions and faq suggestions

### PR DESCRIPTION
**Pull request by SavvyShah**
Documentation updated after adding featured suggestions and FAQ suggestinos support for Vue and React

[Reactivesearch PR](https://github.com/appbaseio/reactivesearch/pull/2221)

### CODE REVIEW

**Review by mohdashraf010897 — APPROVED** [review]
@SavvyShah Resolve the conflicts

File: content/docs/reactivesearch/vue/search/SearchBox.md [hunk 1]
```diff
@@ -1258,4 +1312,49 @@ The following events to the underlying `input` element:
...
+	<SearchBox
+	    title="SearchBox"
+	    dataField={['original_title', 'original_title.search']}
+	    componentId="BookSensor"
+	    innerClass={{
+            'section-label': 'section-label',
+	    	'active-suggestion-item': 'active-test-suggestion',
+	    	'suggestion-item': 'test-suggestion',
+	    }}
+	    enableFeaturedSuggestions
```
Comment (mohdashraf010897) [inline content/docs/reactivesearch/vue/search/SearchBox.md:1606]: @SavvyShah 

- `searchboxId` prop is missing here. (the change has to happen in React version also)
- Can we add a link to a featuredSuggestions example that we have?

File: content/docs/reactivesearch/vue/search/SearchBox.md [hunk 2]
```diff
@@ -1258,4 +1312,49 @@ The following events to the underlying `input` element:
 
 ## Examples
 
+### SearchBox with default props
+
 <a href="https://reactivesearch-vue-playground.netlify.com/?selectedKind=Search%20Components%2FSearchBox&selectedStory=Basic&full=0&addons=1&stories=1&panelRight=0" target="_blank">SearchBox with default props</a>
```
Comment (mohdashraf010897) [inline content/docs/reactivesearch/vue/search/SearchBox.md]: @SavvyShah We are not using netlify anymore, please change the domain to use vercel.

-> https://reactivesearch-vue.vercel.app/

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]
- searchboxId prop is missing here. (the change has to happen in React version also)
- Can we add a link to a featuredSuggestions example that we have?
- We are not using netlify anymore, please change the domain to use vercel.
-> https://reactivesearch-vue.vercel.app/

**Review by mohdashraf010897 — APPROVED** [review]
@siddharthlatest Would recommend you to have a look at the sentence formulation, probably need  some improvisation on that.

File: content/docs/reactivesearch/react/search/searchbox.md [hunk 3]
```diff
@@ -378,6 +378,15 @@ It can accept the following keys:
...
 ```
 
+
+### enableFAQSuggestions
+
+| Type | Optional |
+|------|----------|
+|  `bool` |   Yes   |
+
+Defaults to `false`. When set to `true`, shows frequent user queries which on click instantly shows the answer for the query. FAQs are set from the dashboard by creating a Searchbox. Hence they require a `searchboxId` to be passed.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/react/search/searchbox.md]:
```suggestion
Defaults to `false`. When set to `true`, show frequently asked user questions as configured via the ReactiveSearch dashboard. When this property is set as `true`, `searchboxId` is required to be passed.
```

File: content/docs/reactivesearch/react/search/searchbox.md [hunk 4]
```diff
@@ -397,6 +406,33 @@ Defaults to `false`. When set to `true`, featured suggestions are returned as su
...
+| Type | Optional |
+|------|----------|
+|  `Object` |   Yes   |
+
+
+Specify additional options for fetching featured suggestions.
+
+It can accept the following keys:
+- sectionLabel: `string` Label of the division under which all FAQ suggestions are shown
+- size: `number` maximum number of faq suggestions fetched per section.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/react/search/searchbox.md]:
```suggestion
- size: `number` maximum number of FAQ suggestions to be fetched per section
```

**Review by siddharthlatest — APPROVED** [review]

**Review by siddharthlatest — APPROVED** [review]

**Pull request merged**